Makefile

```diff
@@ -0,0 +1,53 @@
+export CQ_PROVIDER_DEBUG=1
+export CQ_REATTACH_PROVIDERS=.cq_reattach
+
+# install the latest version of CQ
+install-cq:
+	curl -L https://github.com/cloudquery/cloudquery/releases/latest/download/cloudquery_Darwin_x86_64 -o cloudquery
+	chmod +x ./cloudquery
+
+install-cq-mac:
```

Comment:
what's the difference between `install-cq`, `install-cq-mac` (twice). I think we can get the OS and the arch automatically and put it in an env?

Reply:
If we can't, how about mac-m1? Or how about this:

```
install-cq:
@echo Go to https://docs.cloudquery.io/install-instructions and do the thing there please
```

(ie. I'm questioning whether it's needed to have it here)

Reply:
I left `install-cq` target in there because that is what I use. I would love to support automatically detecting arch and os.

Reply:
I added the instructions under this target. When we can support dynamic detection we can replace it with the automated install